SwinjectAutoregistration.xcodeproj/project.pbxproj

```diff
@@ -874,6 +876,7 @@
 				);
 				INFOPLIST_FILE = "$(SRCROOT)/Sources/Info.plist";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.swinject.SwinjectAutoregistration-iOS";
+				SWIFT_VERSION = 5.0;
```

Comment:
You might want to keep the version at 4.2 in order to be compatible with old OS versions. (Hence the failing tests)